### ISSUE TITLE
Windows tiled and snapped to desktop edges get opacity

### ISSUE THREAD

**Issue by sphh** [linuxmint/cinnamon#10891]
```
 * Cinnamon version (cinnamon --version): 5.2.7
   - Please specify if you are using the daily builds PPA
     (https://launchpad.net/~linuxmint-daily-build-team/+archive/ubuntu/daily-builds): No
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...): Mint 20.3
 * 32 or 64 bit: 64 bit
 ```

**Issue**
I have `System Settings > Window Tiling and Snapping` enabled with the following settings:
```
Tiling HUD visibility threshold: 25
Modifier to use for toggling between tile and snap mode: Control
Maximize, instead of tile, when dragging a window to the top edge: NO
Show snap on-screen-display: YES
Show tile heads-up-display: YES
Legacy window snapping (hold <Shift> while dragging a window): NO
```

In `System Settings > Windows` I have:
```
Action on title bar with mouse scroll: Adjust opacity
Minimum opacity (%): 30
```

**Steps to reproduce**
With your mouse grab a window by its title bar and move it to any of the desktop edges. The new tiled and snapped size is shown. Let go the mouse button: The window is tiled and snapped to the desktop edge _and_ it is opacified.

**Expected behaviour**
The window should not get opacified (or there should be a setting controlling this behaviour, possibly with a specified opacity value).

**Other information**
This started to happen after one of the last updates (but unfortunately I cannot remember which one).

**Comment by sphh** [comment]
@mtwebster: Any idea, why that happens?

**Comment by mtwebster** [comment]
Are you saying that when you make a window transparent, then tile it, it loses that transparency? I can't reproduce anything like this (or the reverse) on a 20.3 virtual machine.

Do you have any Cinnamon extensions that might be interfering? I know there have been some that modified window transparency in various ways.

Does it happen both with and without window effects enabled?

**Comment by sphh** [comment]
No, the window is opaque (not transparent) and I tile it (by snapping it to any desktop edge) – it becomes transparent.

In the "Effects" system settings, I have "Window effects" and "Unmaximizing and tiling windows" enabled.If I disable the "Unmaximizing and tiling windows" setting, the window stays opaque.

I then checked each extension (thanks for the tip!) and found the problematic one: [Opacify Windows](https://cinnamon-spices.linuxmint.com/extensions/view/77). Equipped with this knowledge, I also found this behaviour mentioned by Zhao Zuohong in the discussion section. There are also two bugs:
- https://github.com/linuxmint/cinnamon-spices-extensions/issues/326
- https://github.com/linuxmint/cinnamon-spices-extensions/issues/31 (closed because of missing information)

I guess, this issue here can be closed now.

Thanks for your help!
